em_workflows/utils/flow.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+Utilities workflows:
+-------------------
+These workflows are created simply to aid developers.
+
+- Use #1: Run the tests regularly.
+  Test input files can be of big size which can be hard to manage. Since most of the tests are normally executed on
+  HPC over local machine, it is deemed okay have a prefect scheduler to run the tests for us.
+"""
+from pathlib import Path
+import subprocess
+from time import time
+
+from prefect import flow, task
+from prefect.artifacts import create_markdown_artifact
+
+from em_workflows.config import Config
+from em_workflows.utils import utils
+
+RECENT_FILE = "coverage.svg"
+RECENT_HOUR = 1.0
+
+
+@task
+def publish_artifact(report: str):
+    return create_markdown_artifact(
+        key="pytest-cov-report",
+        markdown=report,
+        description="Pytest Coverage Report",
+    )
+
+
+@task
+def is_run_recently():
+    """
+    Checks if the test has been run recently using locally saved file
```

Comment:
Why run this check?

Reply:
When I started, I was not thinking about a scheduler. Now looking at it, it should be removed imo.